src/main/java/net/dries007/tfc/network/PacketStackFood.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.network;
+
+import io.netty.buffer.ByteBuf;
+import net.dries007.tfc.TerraFirmaCraft;
+import net.dries007.tfc.api.capability.food.CapabilityFood;
+import net.dries007.tfc.api.capability.food.IFood;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.inventory.Slot;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessageHandler;
+import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.List;
+
+public class PacketStackFood implements IMessage
+{
+    private int slotNumber;
+
+    public PacketStackFood()
```

Comment:
```suggestion
    @SupressWarnings("unused")
    @Deprecated
    public PacketStackFood()
```
Helps avoiding IDE warnings for not using it (since it's called through forge internals), and accidental uses of it (since it's a common mistake to call this constructor instead of passing in the slot number)